src/decoder.js

```diff
@@ -1,25 +1,29 @@
+import consensysDecode from 'abi-decoder';
+
 /**
  *
  * This will return get function name and parameters out
  * of the trasnaction input attribute if exists.
  *
  */
```

Comment:
You should put the @params in the comments 💃

Reply:
Done